scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+### Inputs
+
+The GVS workflow inputs are specified in the Terra GVS beta workspace [workflow configuration](LINK).
+
+#### Sample data input
+
+The GVS workflow takes in reblocked single sample gVCF files and their corresponding index files as `input_vcfs` and `input_vcf_indexes`, respectively. While the GVS workflow takes in up to 100,000 single sample gVCF files as input, only datasets of up to 10,000 files are being used for beta testing.
+
+The [Terra GVS beta workspace](LINK) has been configured with example reblocked gVCF files that you can use to test the workflow. To test the workflow on your own sample data, you’ll need to create an additional data table in your workspace that specifies the locations of the files in the cloud. For step-by-step instructions on creating data tables in Terra, see [How to make a data table from scratch or a template](https://support.terra.bio/hc/en-us/articles/6197368140955). If your data is stored locally, you can upload it to your workspace bucket using the Data Uploader in Terra by following the instructions in the article, [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411). You’ll also need to ensure that you have a BigQuery dataset and Google project with the correct permissions and that your gVCF files are reblocked, as described in the [Workflow requirements](#workflow-requirements) section above.
+
+#### Input descriptions
+
+The table below describes the GVS workflow input variables:
+
+| Input variable name | Description | Type |
+| --- | --- | --- |
+| dataset_name | String describing the name of the BigQuery dataset. | String |
+| project_id | String describing the name of the Google project that contains the sample data. | String |
+| external_sample_names | Strings describing the unique sample IDs in the Terra data model. | Array of strings |
+| input_vcfs | Cloud paths to the sample gVCF files. | Array of files |
+| input_vcf_indexes | Cloud paths to the sample gVCF index files. | Array of files |
+| callset_identifier | String used as the name of the filter model and as the prefix to the names of the BigQuery extract tables and final joint VCF shards. | String |
+
+## Tasks and tools
+
+The [GVS workflow](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) imports additional WDL scripts that contain the workflow tasks. Links to these additional WDL scripts (subtasks/subworkflows) are provided below.
+
+Overall, the GVS workflow:
+
+1. Imports sample genomes.
+1. Trains the filtering model.
+1. Extracts VCF files.
+
+To see specific tool parameters used in each step of the workflow, select the WDL link in the table; then find the task and view the `command {}` section of the task in the WDL script.
+
+To view or use the exact tool software, see the task's Docker image, which is specified in the task `runtime {}` section as `docker: `.
+
+### 1. Import sample genomes
```

Comment:
If I understand this section correctly, it seems like the workflow is importing the GVCFs (variant info) rather than a genome? Is genome the preferred wording?

Reply:
+1 I think this should be gVCFs